go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/chef/chef-analyze
 go 1.12
 
 require (
+	github.com/afiune/go-chef v0.3.1-0.20191013021240-fcdbdbf2e50c
```

Comment:
Im going to change this to point to our internal fork instead of mine: https://github.com/chef/go-chef